cardano_node_tests/tests/tests_conway/test_committee.py

```diff
@@ -0,0 +1,148 @@
+"""Tests for Conway governance Constitutional Committee functionality."""
+import logging
+import pathlib as pl
+
+import allure
+import pytest
+from cardano_clusterlib import clusterlib
+
+from cardano_node_tests.cluster_management import cluster_management
+from cardano_node_tests.tests import common
+from cardano_node_tests.utils import clusterlib_utils
+from cardano_node_tests.utils import helpers
+from cardano_node_tests.utils import submit_utils
+from cardano_node_tests.utils.versions import VERSIONS
+
+LOGGER = logging.getLogger(__name__)
+DATA_DIR = pl.Path(__file__).parent / "data"
+
+pytestmark = pytest.mark.skipif(
+    VERSIONS.transaction_era < VERSIONS.CONWAY,
+    reason="runs only with Tx era >= Conway",
+)
+
+
+class TestCommittee:
+    """Tests for Constitutional Committee."""
+
+    @pytest.fixture
+    def payment_addr(
+        self,
+        cluster_manager: cluster_management.ClusterManager,
+        cluster: clusterlib.ClusterLib,
+    ) -> clusterlib.AddressRecord:
+        """Create new payment address."""
+        addr = clusterlib_utils.create_payment_addr_records(
+            f"chain_tx_addr_ci{cluster_manager.cluster_instance_num}",
+            cluster_obj=cluster,
+        )[0]
+
+        # Fund source address
+        clusterlib_utils.fund_from_faucet(
+            addr,
+            cluster_obj=cluster,
+            faucet_data=cluster_manager.cache.addrs_data["user1"],
+        )
+
+        return addr
```

Comment:
I suggest to use "cached" fixture like in https://github.com/input-output-hk/cardano-node-tests/blob/master/cardano_node_tests/tests/tests_conway/test_drep.py#L27-L50